### PR TITLE
Add localizations for today and cancel

### PR DESCRIPTION
**Pull request by alogan519**

### CODE REVIEW

File: localize-behavior.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+							}
+						};
+					}
+				}
+			},
+			_computeLanguage: function(resources) {
+				var locale = document.documentElement.lang
+					|| document.documentElement.getAttribute('data-lang-default')
+					|| 'en-us';
+				locale = locale.toLowerCase();
```
Comment (awikkerink) [inline localize-behavior.html]: The `locale` is specified on the d2l-date-picker component, so you should be getting it from there.

Comment (dlockhart) [inline localize-behavior.html]: Agreed. I'm going to build (hopefully this week) a behaviour that works with app-localize that will pull the locale off the document (similar to this code) and resolve it against the available locales exposed by the component. Not ready yet though, but I'll switch this over when it is.

File: localize-behavior.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+								cancel: 'iptal etmek'
+							},
+							'zh': {
+								today: '今天',
+								cancel: '取消'
+							},
+							'zh-tw': {
+								today: '今天',
+								cancel: '取消'
+							}
```
Comment (dlockhart) [inline localize-behavior.html]: We also now support German (DE) and Norwegian (NB).

File: localize-behavior.html (hunk 1 above)
Comment (dlockhart) [inline localize-behavior.html]: @alogan519 was just following what the existing more-less control is doing (I pointed him there). That's arguably OK in the short-term until the library I'm building (see above) is available.

File: localize-behavior.html (hunk 2 above)
Comment (alogan519) [inline localize-behavior.html]: Oh ok, I think I'll add this in later, trying to get this in for Lessons which I think mostly we need just English and French for this week.

File: d2l-date-picker.html [hunk 3]
```diff
@@ -128,13 +134,16 @@
...
+				this.language = this.locale;
 
 				var datePicker = this.$$('vaadin-date-picker-light');
 				var localeOverrides = {
 					monthNames: locale.date.calendar.months.long,
 					weekdays: locale.date.calendar.days.long,
 					weekdaysShort: locale.date.calendar.days.short,
-					firstDayOfWeek: locale.date.calendar.firstDayOfWeek
+					firstDayOfWeek: locale.date.calendar.firstDayOfWeek,
+					today: this.localize('today'),
```
Comment (awikkerink) [inline d2l-date-picker.html]: `localize` is not calculated until `language` and `resources` are set

Comment (awikkerink) [inline d2l-date-picker.html]: You can use a complex observer:

```javascript
observers: [
   '_updateDatePickeri18n(locale, localize)'
],
```

File: d2l-date-picker.html [hunk 4]
```diff
@@ -128,13 +134,16 @@
 
 			_updateDatePickeri18n: function() {
 				var locale = d2lIntl.LocaleProvider(this.locale);
+				this.language = this.locale;
```
Comment (awikkerink) [inline d2l-date-picker.html]: This should be set in another observer, since `language` doesn't depend on `localize`, but `localize` depends on `language`

**Pull request merged**